.github/workflows/rspec.yml

```diff
@@ -0,0 +1,28 @@
+name: RSpec
+
+on:
+  push:
+    branches:
+    - "main"
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Install dependencies
+        run: bundle install
+
+      - name: Run test and linter
+        run: |
```

Comment:
The `|` here is good for multi-line commands. In this case, I think we can inline that one line and skip the pipe altogether.